core/templates/dev/head/exploration_editor/exploration_editor.html

```diff
@@ -257,11 +257,29 @@
         <p>
           Just a few more things before you publish:
         </p>
+        <p ng-if="requireTitleToBeSpecified">
+          Add a Title: what should your exploration be called?
+          <input type="text" class="form-control" ng-model="explorationTitleService.displayed"
+                 placeholder="Music in the 19th Century">
+        </p>
         <p ng-if="requireObjectiveToBeSpecified">
-          Add a goal: what does thie exploration help people to do?
+          Add a goal: what does the exploration help people to do?
           <input type="text" class="form-control" ng-model="explorationObjectiveService.displayed"
                  placeholder="Learn how to ...">
         </p>
+        <p ng-if="requireCategoryToBeSpecified">
+          Add a category: how would you categorize your exploration?
+            <select2-dropdown id="explorationCategory" item="explorationCategoryService.displayed"
```

Comment:
This block should be de-indented by 2. It's on the same 'level' as the preceding text.